Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocCellAttributeRenderingContext.cs

```diff
@@ -0,0 +1,42 @@
+﻿// // This file is part of the re-motion Core Framework (www.re-motion.org)
+// // Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// //
+// // The re-motion Core Framework is free software; you can redistribute it
+// // and/or modify it under the terms of the GNU Lesser General Public License
+// // as published by the Free Software Foundation; either version 2.1 of the
+// // License, or (at your option) any later version.
+// //
+// // re-motion is distributed in the hope that it will be useful,
+// // but WITHOUT ANY WARRANTY; without even the implied warranty of
+// // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// // GNU Lesser General Public License for more details.
+// //
+// // You should have received a copy of the GNU Lesser General Public License
+// // along with re-motion; if not, see http://www.gnu.org/licenses.
+// //
+namespace Remotion.ObjectBinding.Web.UI.Controls.BocListImplementation.Rendering
+{
+  /// <summary>
+  /// Provides forwarding of additional attributes to  <see cref="BocColumnRenderingContext"/>
+  /// </summary>
+  public struct BocCellAttributeRenderingContext<TBocColumnDefinition>
```

Comment:
Naming:
Please change this to two separate structs:
`BocListTitleCellAttributeRenderingContext`
`BocListDataCellAttributeRenderingContext`

Reasoning: makes the API more stable in case we need to introduce arguments in the future that are specific to Title or Data cell. Having two separate struct types for this isn't costing us anything compared to future breaking changes.

Reply:
Sorry, you got there first ;/ @a-ctor mentioned that since there will ever only be one usecase and we already have the `BocDataCell...` and `BocTitleCell...` naming conventions, using `BocCellAttributeRenderingContext` is actually correct.